Framework/Blockhub.Core/Services/Wallet/SearchForAccountsByBalanceWalletCreate.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Blockhub.Services;
+using Blockhub.Wallet;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Blockhub.Wallet
+{
+    public class SearchForAccountsByBalanceWalletCreate<T> : IWalletCreate<T> where T : IBlockchainType
+    {
+        private IWalletCreate<T> WalletCreator { get; }
+        private IAccountCreate<T> WalletManager { get; }
+        private IBalanceRead<T> BalanceReader { get; }
+        private int SearchCount { get; }
+
+        public SearchForAccountsByBalanceWalletCreate(IWalletCreate<T> walletCreator, IAccountCreate<T> walletManager, IBalanceRead<T> balanceReader) :
```

Comment:
Could we do wallet creation and searching for accounts separately? There's a lot of code here for something that seems like it could be a couple helper methods. The class name is a little unclear too, if I had to rename maybe `SearchForAccountsByBalanceAndCreateWallet` or `AccountByBalanceWalletCreator` but I would wonder why I don't just have code like this:

```
Wallet wallet = new Wallet(...);
Accounts[] accounts = Wallet.SearchForAccounts(wallet);
```

Reply:
Ok. I will put something together for this.